updatenewlive.py

```diff
@@ -43,6 +43,9 @@
 		livesetting = jpdbconn.execute('SELECT * FROM live_setting_m WHERE live_track_id = '+str(jptmp[0])+';')
 		livetmp = livesetting.fetchone()
 		while livetmp:
+			# Exclude AC charts
+			if livetmp[0] >= 10000:
+				continue
```

Comment:
Noticed just now, the `continue` here will result in infinite loop.
Need `livetmp = livesetting.fetchone()` before `continue`